src/components/ChattingScreen.js

```diff
@@ -1,31 +1,135 @@
-import React from 'react';
-import styled from 'styled-components';
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
 
-import Header from './Header';
-import MessageSender from './MessageSender';
+import Header from "./Header";
+import MessageSender from "./MessageSender";
 
 export default function ChattingScreen() {
-	const EUNKO = 'https://img.techpowerup.org/200908/eun.png';
-	const COOL = 'https://img.techpowerup.org/200908/NjRiY2JjOGU5YzQz.png';
-
-	const MSGLIST = [
-		{ user: true, content: '안녕하세요 12기 프론트엔드 개발자분들' },
-		{ user: true, content: '저희의 대화를 마음껏 조작해보세요 💌' },
-		{ user: true, content: '상단에 프로필을 눌러서 발신자 변경하면 됩니당~' },
-		{ user: false, content: '안녕 은아' },
-		{ user: false, content: '뭐해 ?' },
-		{ user: true, content: '시원아 넌 최고의 팀장이야' },
-		{ user: false, content: '나도 아니까  ' },
-		{ user: false, content: '그만 말해줘도 돼' },
-		{ user: true, content: 'ㅠㅠ' },
-	];
-
-	return (
-		<Wrapper>
-			<Header></Header>
-			<MessageSender></MessageSender>
-		</Wrapper>
-	);
+  const EUNKO = "https://img.techpowerup.org/200908/eun.png";
+  const COOL = "https://img.techpowerup.org/200908/NjRiY2JjOGU5YzQz.png";
+
+  const MSGLIST = [
+    { user: true, content: "안녕하세요 12기 프론트엔드 개발자분들" },
+    { user: true, content: "저희의 대화를 마음껏 조작해보세요 💌" },
+    { user: true, content: "상단에 프로필을 눌러서 발신자 변경하면 됩니당~" },
+    { user: false, content: "안녕 은아" },
+    { user: false, content: "뭐해 ?" },
+    { user: true, content: "시원아 넌 최고의 팀장이야" },
+    { user: false, content: "나도 아니까  " },
+    { user: false, content: "그만 말해줘도 돼" },
+    { user: true, content: "ㅠㅠ" },
+  ];
+
+  const [messageList, setMessageList] = useState(MSGLIST); //intial state값을 MSGLIST로
+  const [newMessage, setNewMessage] = useState({
+    user: false,
+    content: "",
+  });
+
+  const changeInputMessage = (e) => {
+    const { name, value } = e.target; //name으로 구별, value는 값 자체
+    setNewMessage({
+      ...newMessage,
+      [name]: value,
+    });
+  };
+
+  const addMessage = (e) => {
+    e.preventDefault();
+    if (newMessage.content) {
+      setMessageList([...messageList, newMessage]);
+    } else {
+      alert("메시지를 입력하세요");
+    }
+    setNewMessage({ ...newMessage, content: "" });
+  };
+
+  const changeUser = (e) => {
+    e.preventDefault();
+    if (!newMessage.user) {
+      //현재 시원님인 상태
+      return setNewMessage({
+        user: true, //누르면 고은님으로 change
+        content: newMessage.content,
+      });
+    } else {
+      return setNewMessage({
+        user: false,
+        content: newMessage.content,
+      });
+    }
+  };
+
+  useEffect(() => {
+    window.scrollBy(0, window.innerHeight);
```

Comment:
`window.innerHeight` 보다는 `document.body.scrollHeigh`t을 이용하시는게 좋을 것 같아요! 
메시지를 길게 추가하다 보면` window.innerHeigh`t적용시에는, 맨 밑까지 한번에 스크롤되지 않는걸 확인할 수 있으실거에요ㅎㅎ
```suggestion
    window.scrollBy(0, document.body.scrollHeigh);
```

Reply:
넵 수정할게요